src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 div:first-child,
```

```suggestion
.stars--1 :first-child,
```
There is no need to increase the specificity. Fix all cases